frontend/src/components/createProjectForm.tsx

```diff
@@ -0,0 +1,282 @@
+import useTranslation from "next-translate/useTranslation";
+import { Badge, Col, Row } from "react-bootstrap";
+import styles from "../styles/createProjectForm.module.css";
+import { Field, Form, Formik } from "formik";
+import useSWR from "swr";
+import apiPaths from "../properties/apiPaths";
+import { getAllUsersFromPage } from "../api/calls/userCalls";
+import { capitalize } from "../utility/stringUtil";
+import { ProjectCreationProps, ProjectCreationValues } from "../handlers/createProjectSubmitHandler";
+import { IUser } from "../api/entities/UserEntity";
+import { getAllSkillTypesFromPage } from "../api/calls/skillTypeCalls";
+import { ISkillType } from "../api/entities/SkillTypeEntity";
+import { ChangeEvent, useState } from "react";
+import Image from "next/image";
+
+export const CreateProjectForm = (props: ProjectCreationProps) => {
+    let userResponse = useSWR(apiPaths.users, getAllUsersFromPage);
```

Comment:
Do we need a search for a user within an edition here?

Reply:
probably, yes

Reply:
Please report this in the backend endpoint issue.

Reply:
I did this.